syft/frameworks/torch/tensors/interpreters/Polynomial.py

```diff
@@ -0,0 +1,57 @@
+from syft.frameworks.torch.tensors.interpreters.add import AdditiveSharingTensor
+import torch
+
+
+class PolynomialTensor(AdditiveSharingTensor):
```

Comment:
IMO - PolynomialTensor (PT) should not be a subclass of AdditiveSharingTensor (AST) because while AST will use PT, PT will also be used in an identical way by other tensors (such as Zero Knowledge Tensor). Thus, we want PT to be a tensor on its own (extending AbstractTensor) with a .child which is assumed to be an integer type. The main unit tests can then proceed with IntegerTensor underneath with some light unit testing of compatability with AdditiveSharingTensor